url79d4f7c7ed_schanq_Live_Algorithm_tgz-pJ8-algorithms_and_systems_AI_Evolutionary_GeneticAlgorithmJ8/src/expert_system/Feel.java

```diff
@@ -0,0 +1,213 @@
+			fitRhythms.add(Rhythmic_Block.SEMI_QUAVERS4_THIRD_AND_FORTH_OFF);
+			fitRhythms.add(Rhythmic_Block.SEMI_QUAVERS4_THIRD_OFF);
+			fitRhythms.add(Rhythmic_Block.SEMI_QUAVERS8);
+			fitRhythms.add(Rhythmic_Block.SEMI_QUAVERS8_FIRST_SECOND_FORTH_AND_FIFTH_OFF);
+			fitRhythms.add(Rhythmic_Block.SEMI_QUAVERS8_FIRST_THIRD_FIFTH_AND_SEVENTH_OFF);
+			fitRhythms.add(Rhythmic_Block.SEMI_QUAVERS8_SECOND_FIFTH_SEVENTH_AND_EIGTH_OFF);
+			fitRhythms.add(Rhythmic_Block.SEMI_QUAVERS8_SECOND_FORTH_SIXTH_AND_EIGTH_OFF);
+			fitRhythms.add(Rhythmic_Block.SEMI_QUAVERS8_SECOND_THIRD_FIFTH_AND_SIXTH_OFF);
+			fitRhythms.add(Rhythmic_Block.SEMI_QUAVERS8_THIRD_SIXTH_AND_EIGTH_OFF);
+			fitRhythms.add(Rhythmic_Block.TRIPLET_SEMI_QUAVERS_FULL);
+			fitRhythms.add(Rhythmic_Block.TRIPLET_SEMI_QUAVERS_VAR1);
+			fitRhythms.add(Rhythmic_Block.TRIPLET_SEMI_QUAVERS_VAR2);
+			fitRhythms.add(Rhythmic_Block.TRIPLET_SEMI_QUAVERS_VAR3);
+			fitRhythms.add(Rhythmic_Block.DEMI_SEMI_QUAVERS8);
+			fitRhythms.add(Rhythmic_Block.DEMI_SEMI_QUAVERS8_FIRST_SECOND_FORTH_AND_FIFTH_OFF);
+			fitRhythms.add(Rhythmic_Block.DEMI_SEMI_QUAVERS8_FIRST_THIRD_FIFTH_AND_SEVENTH_OFF);
+			fitRhythms.add(Rhythmic_Block.DEMI_SEMI_QUAVERS8_SECOND_FIFTH_SEVENTH_AND_EIGTH_OFF);
+			fitRhythms.add(Rhythmic_Block.DEMI_SEMI_QUAVERS8_SECOND_FORTH_SIXTH_AND_EIGTH_OFF);
+			fitRhythms.add(Rhythmic_Block.DEMI_SEMI_QUAVERS8_SECOND_THIRD_FIFTH_AND_SIXTH_OFF);
+			fitRhythms.add(Rhythmic_Block.DEMI_SEMI_QUAVERS8_THIRD_SIXTH_AND_EIGTH_OFF);
+			fitRhythms.add(Rhythmic_Block.TRIPLET_DEMI_SEMI_QUAVERS_FULL);
+			fitRhythms.add(Rhythmic_Block.TRIPLET_DEMI_SEMI_QUAVERS_VAR1);
+			fitRhythms.add(Rhythmic_Block.TRIPLET_DEMI_SEMI_QUAVERS_VAR2);
+			fitRhythms.add(Rhythmic_Block.TRIPLET_DEMI_SEMI_QUAVERS_VAR3);
+			
+		}
+		else if(name == "SWUNGWALK"){
+			
+			fitRhythms.add(Rhythmic_Block.MINIM1);
+			fitRhythms.add(Rhythmic_Block.CROTCHETS1);
+			fitRhythms.add(Rhythmic_Block.CROTCHETS2);
+			fitRhythms.add(Rhythmic_Block.CROTCHET_SWUNG_T1);
+			fitRhythms.add(Rhythmic_Block.CROTCHET_SWUNG_T2);
+			fitRhythms.add(Rhythmic_Block.CROTCHET_SWUNG_T3);
+			fitRhythms.add(Rhythmic_Block.CROTCHET_SWUNG_T4);
+			fitRhythms.add(Rhythmic_Block.TRIPLET_CROTCHETS_FIRST_OFF);
+			fitRhythms.add(Rhythmic_Block.TRIPLET_CROTCHETS_FULL);
+			fitRhythms.add(Rhythmic_Block.TRIPLET_CROTCHETS_SECOND_OFF);
+			fitRhythms.add(Rhythmic_Block.TRIPLET_CROTCHETS_THIRD_OFF);
+			fitRhythms.add(Rhythmic_Block.QUAVERS_SWUNG_T1);
+			fitRhythms.add(Rhythmic_Block.QUAVERS_SWUNG_T2);
+			fitRhythms.add(Rhythmic_Block.QUAVERS_SWUNG_T3);
+			fitRhythms.add(Rhythmic_Block.QUAVERS_SWUNG_T4);
+			fitRhythms.add(Rhythmic_Block.TRIPLET_DEMI_SEMI_QUAVERS_VAR3);
+			fitRhythms.add(Rhythmic_Block.TRIPLET_QUAVERS_FIRST_OFF);
+			fitRhythms.add(Rhythmic_Block.TRIPLET_QUAVERS_FULL);
+			fitRhythms.add(Rhythmic_Block.TRIPLET_QUAVERS_SECOND_OFF);
+			fitRhythms.add(Rhythmic_Block.TRIPLET_QUAVERS_THIRD_OFF);
+				
+		}
+		else if(name == "STRAIGHTWALK"){
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

**Problem**: An equality operator (== or !=) is used to compare strings, which matches strings based on address.

**Fix**: Do content comparison using the equals() method to compare the values of the strings.